### DISCUSSION TITLE
V5 | Language files

### DISCUSSION THREAD

**Discussion by PandaKeny** [SudhanPlayz/Discord-MusicBot#992] [Ideas]
Like the title, I want a separate language file and people can translate bots in many different languages.

**Comment by JotaroKujo0525** [comment]
Hm, ig not really necessary for the whole v5 code as english is.... you know, the language that almost all people know nowadays.
Also, who will translate? Well, we can't also use Google translator as its not accurate

**Comment by JotaroKujo0525** [comment]
It is indeed doable, but not the type that we really need

**Comment by PandaKeny** [comment]
Translators can be all those who want bot translation can support, I know English is a popular language but there will definitely be people who are not good at English, language files to be more friendly to people and the language of their country.

**Comment by JotaroKujo0525** [comment]
@DarrenOfficial @SudhanPlayz your thoughts?

**Comment by JotaroKujo0525** [comment]
Also I'll convert this to an idea discussion to not mix up with other issues

**Comment by DarrenOfficial** [comment]
this has already been discussed, it will be added soon.

**Reply by JotaroKujo0525** [reply]
kk

**Reply by PandaKeny** [reply]
This seems like you and I talked about it in the lavalink issue =)))

**Reply by PandaKeny** [reply]
> this has already been discussed, it will be added soon.

Has it been done yet?

**Comment by Saud-97** [comment]
Weblate has a nice integration with Github for translation managment https://hosted.weblate.org/projects/
Example: https://github.com/tachiyomiorg/tachiyomi/pull/7490

**Comment by KFR98** [comment]
I am recently using the bot, but would like to contribute in translating it into Italian.
